core/profile/extractor.py

```
@@ -0,0 +1,150 @@
+
+from .config import ProfileExtractionConfig
+
+logger = logging.getLogger(__name__)
+
+# ------------------------------------------------------------
+# 提取 Prompt
+# ------------------------------------------------------------
+
+EXTRACTION_PROMPT = """你是一个用户画像提取器。从以下对话消息中提取用户的事实性标签。
+
+规则：
```

Ruff (0.15.12)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)